corehq/apps/analytics/utils/partner_analytics.py

```diff
@@ -143,7 +143,7 @@ def get_csv_details_for_partner(contact, year, month):
                     domain,
                     data_point.first().value,
                 ])
-    return headers, body
+    return headers, sorted(body)
```

Comment:
@biyeun I changed this to stabilized the order of returned records since SQL does not guarantee any particular order by default. Does this seem like a reasonable modification to you?

Reply:
PR's separately in https://github.com/dimagi/commcare-hq/pull/32167